src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
 
+        Machine[] machines = {truck, bulldozer, excavator};
+        for (Machine mach : machines) {
```

Comment:
```suggestion
        for (Machine machine: machines) {
```
don't use short versions of words